src/commands/init.js

```diff
@@ -96,6 +96,25 @@ async function init() {
     }
   }
 
+  progress = ora('Adding new entries to .gitignore').start();
```

Comment:
How about we show this only if: a) `gitignore` exists and it doesn't contain `haul` yet? Otherwise it feels like unnecessary output.